oat-sa/extension-tao-itemqti
# feat: use backend generated item identifier if provided

related to https://oat-sa.atlassian.net/browse/ADF-1801

### Description

Use backend-provided item identifier if supplied. Use frontend-generated one if not.

### How to test

create new item. enter its authoring, check the identifier is the same as with provided by getItemData backend call
